### ISSUE TITLE
One small question

### ISSUE THREAD

**Issue by ooobsidian** [seongmin-kye/meta-SR#7]
For the "Unseen speaker identification," that means the speaker in the test set isn't seen in the training set? Thank you. @seongmin-kye
